frontend/package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "@expo/vector-icons": "^12.0.0",
     "@react-native-async-storage/async-storage": "^1.13.4",
+    "@react-native-community/datetimepicker": "^3.0.9",
```

Comment:
You should use version 3.0.4 as that is the version compatible with our currently installed expo package version.

![image](https://user-images.githubusercontent.com/3270324/108608594-eaf51000-737c-11eb-91b6-cc0300c3e3a6.png)

Reply:
agreed, I'll fix that.